src/qml/PositioningInformationView.qml

```diff
@@ -19,15 +19,15 @@ Rectangle {
 
   property double antennaHeight: NaN
   property double rowHeight: 30
-  property color backgroundColor: Theme.mainBackgroundColor
+  property color backgroundColor: "transparent" 
   property color alternateBackgroundColor: Theme.positionBackgroundColor
   property color textColor: positionSource.currentness ? Theme.mainTextColor : Theme.secondaryTextColor
 
-  height: grid.rows * positioningInformationView.rowHeight
+  property real contentHeight: grid.rows * positioningInformationView.rowHeight
   width: parent.width
   anchors.margins: 20
 
-  color: Theme.mainBackgroundColor
+  color: "#dd" + Theme.mainBackgroundColor.toString().slice(1)
```

Comment:
I think we'd be better of adding a Theme.mainBackgroundColorSemiOpaque instead of this more obscure (and prone to be inconsistent) method.

Reply:
I was initially hesitant to modify the `Theme.qml` file, but since I've been granted permission, I'll to make the necessary changes.

Reply:
any suggestion for name? `mainBackgroundColorTransparent` ? `transparentMainBackground` ? What do you think?

Reply:
mainBackgroundColorSemiOpaque to match other color names please